src/status_im/extensions/core.cljs

```diff
@@ -148,15 +150,16 @@
    {::arithmetic m}))
 
 (defn button [{:keys [on-click]} label]
-  [button/secondary-button {:on-press #(re-frame/dispatch (on-click {}))} label])
+  [button/secondary-button (when on-click {:on-press #(re-frame/dispatch (on-click {}))}) label])
 
 (defn input [{:keys [on-change placeholder]}]
-  [react/text-input {:placeholder placeholder
-                     :style {:width "100%"}
-                     :on-change-text #(re-frame/dispatch (on-change {:value %}))}])
+  [react/text-input (merge {:placeholder placeholder
+                            :style {:width "100%"}}
```

Comment:
why `width "100%"` but not `flex 1` ?

Reply:
Don't quite remember.. But it's not new from this PR